### DISCUSSION TITLE
Anyway to ignore all errors when running with python script?

### DISCUSSION THREAD

**Discussion by longtnguyen921510** [mitmproxy/mitmproxy#5612] [Q&A] [answered]
Hello, is there anyway to have all the connection errors and such not show in the terminal when I run mitmweb -s pathToPythonScript?

Basically all I want to see is the output from my python script.

**Comment by Prinzhorn** [comment, marked as answer]
`--quiet` and then use vanilla `print()` and not the `logging` for your script

**Reply by longtnguyen921510** [reply]
thank you so much!
